src/main/java/org/apache/taglibs/standard/util/XmlUtil.java

```diff
@@ -69,7 +69,14 @@
     private static final DocumentBuilderFactory PARSER_FACTORY;
     private static final SAXTransformerFactory TRANSFORMER_FACTORY;
     private static final SAXParserFactory SAXPARSER_FACTORY;
+
+    private static boolean secureProcessing;
+
     static {
+
+        String secure = System.getProperty("org.apache.taglibs.secureXmlProcessing");
+        secureProcessing = ! (secure != null && secure.equalsIgnoreCase("false"));
```

Comment:
I don't see such method in JDK. Are you suggesting it should be set to false by default?